src/main/kotlin/com/exactpro/th2/conn/amqp/BoxMain.kt

```diff
@@ -68,12 +74,28 @@ fun main(args: Array<String>) {
         )
 
         val rawRouter: MessageRouter<RawMessageBatch> = factory.messageRouterRawBatch
-        configuration.parameters.forEach{connParameters ->
+        val aliasToService = HashMap<String, ConnService>()
+        val counters: Map<Direction, Counter> = EnumMap<Direction, Counter>(Direction::class.java).apply {
+            put(Direction.FIRST, Counter.build().apply {
+                name("th2_conn_incoming_msg_quantity")
+                labelNames("session_alias")
+                help("Quantity of incoming messages to conn")
+            }.register())
+            put(Direction.SECOND, Counter.build().apply {
+                name("th2_conn_outgoing_msg_quantity")
+                labelNames("session_alias")
+                help("Quantity of outgoing messages from conn")
+            }.register())
+        }
```

Comment:
Additional counters is redundant because last common already includes special metrics to monitor incoming / outgoing messages
https://exactpro.atlassian.net/browse/TH2-1713